### ISSUE TITLE
Use of .css

### ISSUE THREAD

**Issue by Masclins** [kenan-rhoton/card-game#128]
Right now we are not using any .css file. That means that all style is on the .pug files.

I think it's more than beyond discussion that it's better to have all style guidelines together in a .css file, avoiding redundancy and useless parameters (see #118), and making it easy to change anything style-related.

**Comment by kenan-rhoton** [comment]
I don't think we have any styles anywhere, whether `.pug` or otherwise. All we have are the use of Bootstrap components, which refer to functionality more than style, and are thus fine in the `.pug` files, except for maybe the card highlighting.

However, we will need to use CSS if we want it to look nice at all...

**Comment by pernilsalat** [comment]
There is also less or sas (super awesome stylesheet), that offers some dinamism and 'logic' to css.
